assets/dev/scss/frontend/_swiper-customisations.scss

```diff
@@ -1,4 +1,5 @@
-.elementor-element {
+.elementor-element,
+.elementor-lightbox {
```

Comment:
is it work doing this?
```
.elementor {
	&-element,
	&-lightbox {
		...
	}
}
```
does it make it more ore less readable/efficient - is up to you

Reply:
@StuartElementor Nice. Done.